src/main/kotlin/gg/flyte/christmas/listeners/HousekeepingEventListener.kt

```diff
@@ -312,4 +340,16 @@ class HousekeepingEventListener : Listener, PacketListener {
 
         standardMenu.open(false, player)
     }
+
+    private val lobbyRegion = MapRegion(
+        MapSinglePoint(545, 100, 500),
+        MapSinglePoint(575, 120, 535)
+    )
+
+    private fun shouldProcessSnowballEvent(player: Player): Boolean {
+        return eventController().currentGame == null &&
```

Comment:
This is the only condition that will validate the event. The others are either implicitly known, or never changes (ex. gamemode)